src/editor/Renderer.cpp

```diff
@@ -397,7 +401,13 @@ void Renderer::renderLoop() {
 		if (glfwGetTime( ) - lastTitleTime > 0.1)
 		{
 			lastTitleTime = glfwGetTime( );
-			glfwSetWindowTitle(window, std::string(std::string("bspguy - ") + getMapContainingCamera()->map->path).c_str());
+			if (BspRenderer * tmpRend = getSelectedMap())
+			{
+				if (tmpRend->map && tmpRend->map->path.size())
+				{
+					glfwSetWindowTitle(window, std::string(std::string("bspguy - ") + tmpRend->map->path).c_str());
+				}
+			}

```

Comment:
this only needs to be set on map loads

Reply:
@wootguy yes this code change title if map opened or changed